src/scss/_functions.scss

```diff
@@ -47,7 +47,7 @@
 @function oFontsGetFontFamilyWithFallbacks($family) {
 	@if $family == 'Clarion' {
 		@warn "Clarion has been removed, use Georgia instead";
-		@return 'Georgia';
+		@return $o-fonts-serif;
```

Comment:
A mixin cannot `@return` anything, not sure why this is here in the first place. This should not output anything.